addons/website_livechat/static/tests/tours/website_livechat_common.js

```diff
@@ -31,10 +32,10 @@ LivechatButton.include({
                             payload: {
                                 id: self._livechat._id,
                                 message: {
-                                    id: -1,
+                                    id: self._messages.length + 1,
                                     author_id: [0, 'Website Visitor Test'],
                                     email_from: 'Website Visitor Test',
-                                    body: '<p>' + message.content + '</p>',
+                                    body: utils.Markup('<p>' + message.content + '</p>'),
```

Comment:
What is `message.content`?

Reply:
It's just a little thing I noticed while running the testing tour, it currently prints the p tags into the conversation, which does not look very nice when the tour fails and you receive the screenshot.

The "message.content" in that context is the text input we send during the tour steps.
Since it's a testing tour, I did not check deeply (I mean maybe you could use it against yourself while testing but I don't see the point).

If it's annoying to you, I can 100% remove it, it's not really important.

Reply:
Ah no it's fine I had not noticed that this is some tour support stuff.